doc/snek.adoc

```diff
@@ -292,31 +298,33 @@ hello, world
 ----
 
 There's lots of stuff going on here. First, we see how to declare a
-function by using the `def` keyword, followed by the name of the
-function, followed by the “arguments” in parentheses (we'll talk about
-arguments in the next section, for now just type `()`). After the arguments
-there's a colon.
-
-Colons appear in several places in Snek and (outside of dictionaries)
-are used in the same way. After a colon, Snek expects to see a list of
-statements. The usual way of including a list of statements is to type
-them, one per line, indented from the line containing the colon by a
-few spaces. The number of spaces doesn't matter, but each line has to
-use the same indentation. When you're done with the list of
-statements, you enter a line with the old indentation level.
-
-While entering a long compound statement like this, the command
+function: by using the `def` keyword, followed by the name of the
+function, followed by the “arguments” in parentheses. (We'll talk about
+arguments in the next section, for now just type `()`.)
+There's a colon after the arguments.
+
+Colons appear in several places in Snek and (outside of
+dictionaries) are used in the same way. After a colon, Snek
+expects to see a list of statements. The usual way of
+including a list of statements is to type them, one per
+line. Each statement in the list is indented from the line
+containing the colon by a few spaces. The number of spaces
+doesn't matter, but each line has to use the same
+indentation. When you're done with the list of statements,
+finish by going back to the old indentation level.
+
+While entering a list of statements, the command
 processor will prompt with `+` instead of `>` to let you know that
 it's still waiting for more input before it does anything. It's the
 “line with the old indentation level” that displays the second
-`+`. Hitting enter on that line ends the list of statements for the
+`>`. Hitting enter on that line ends the list of statements for the
```

Comment:
Hrm. this is unclear (clearly). What I'm trying to explain is why you have to hit return twice after entering a compound statement in the repl:

> if true:
+  print('yup')
+ 
yup
>